docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -276,3 +276,62 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## SWAP
+
+Endpoints to make SWAP transactions
+
+### List of tokens available for SWAP
+
+`GET /v1/swap/tokens`
+
+#### Path parameters
+
+* `projectId` - is the project identifier
+* `chainId` - filter by chain. CAIP-10 format.
+
+#### Success response body:
+
+* `tokens` - list of objects which define available tokens:
+    * `name` - token name.
+    * `symbol` - token symbol.
+    * `address` - contract address of token.
+    * `chainId` - ID of chain in CAIP-10 format.
+    * `decimals` - decimals for amount supported by a given token.
+    * `iconUrl` - URL of the token icon.
```

Comment:
can any of these be undefined? e.g. iconUrl?

Reply:
Nope, they all expected to be fulfilled.